rapyuta_io/clients/project.py

```diff
@@ -18,8 +16,6 @@ class Project(ObjBase):
     """
     Project is an organizational unit and all the resources must belong to a Project.
```

Comment:
question: Are we keeping the Project class?

Reply:
Kept the model only as its used in user group. Updated it as per v2 response.